leader-election-impl/src/main/java/com/palantir/paxos/PaxosLatestRoundVerifierImpl.java

```diff
@@ -51,7 +55,12 @@ public PaxosQuorumStatus isLatestRound(long round) {
     }
 
     private boolean acceptorAgreesIsLatestRound(PaxosAcceptor acceptor, long round) {
-        return round >= acceptor.getLatestSequencePreparedOrAccepted();
+        try {
+            return round >= acceptor.getLatestSequencePreparedOrAccepted();
+        } catch (Exception e) {
+            log.info("latest sequence retrieval failed for acceptor: {}", acceptor, e);
```

Comment:
I don't think the PaxosAcceptor has a useful toString so I don't think this is useful to log.  If it is useful it needs to be wrapped in `SafeArgs.of(...)` to conform to spec.